pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -19,12 +18,15 @@ func runDbCommand(command func(commandLine utils.CommandLine, sqlStore *sqlstore
 		cmd := &utils.ContextCommandLine{Context: context}
 
 		cfg := setting.NewCfg()
+
 		cfg.Load(&setting.CommandLineArgs{
 			Config:   cmd.String("config"),
 			HomePath: cmd.String("homepath"),
-			Args:     flag.Args(),
+			Args:     context.Args().Tail(),
```

Comment:
I haven't really had time to figure out exactly why. But `context.Args().Tail()` does not work for me when executing `grafana-cli admin data-migration encrypt-datasource-passwords`. I'm *guessing* its related to args count but I dont know right now.

 Given we just 

```go
flag.Parse()
flag.Args()
```
in `grafana-server`. perhaps we should stick to that here as well?

Reply:
It's exactly that - is related to the position of the arguments. It's the main downside to the wrapper approach, we'll always need to keep it up to date with whichever commands we add.

My last commit goes in a slightly different direction - with this branch you should have no problem using the `grafana-cli` directly as is the original binary + the fix in the first commit. The wrapper now has an explicit `.wrapper` suffix.

That being said, I'm happy to add a bit more logic to the wrapper to take into account sub-commands. However, it feels a bit strange that `data-migration` sub-command only has 1 command underneath it, could it be that the plan is to add more of these in the future? If not, I'd be in favour of deleting the third level to bring it inline to the `admin` command - the downside? Backwards compatibility unless we create an alias.

Reply:
My last commit removes the level of indentation - if we decide to move in a different direction I'm happy to take it out.